src/store/assets/tokens.ts

```diff
@@ -58,25 +62,51 @@ export const assetsState = atom(get =>
   get(waitForAll(transformAssets(get(accountDataState)?.balances).map(assetItemState)))
 );
 
+export const assetsUnanchoredState = atom(get =>
+  get(waitForAll(transformAssets(get(accountDataState)?.unanchoredBalances).map(assetItemState)))
+);
+
 export const transferableAssetsState = atom(get =>
   get(assetsState)?.filter(asset => asset.canTransfer)
 );
 
-export const fungibleTokensState = atom(get => {
-  const assets = get(assetsState);
-  return assets?.filter(asset => asset.type === 'ft');
-});
-export const nonFungibleTokensState = atom(get => {
-  const assets = get(assetsState);
-  return assets?.filter(asset => asset.type !== 'nft');
-});
+const calculateBalanceByType = (assetType: string) =>
```

Comment:
i'd make this an atomFamily
```suggestion
const calculateBalanceByType = atomFamily((assetType: string) =>
```

Reply:
Sure !